contracts/sol6/KyberHintHandler.sol

```diff
@@ -311,6 +317,20 @@ abstract contract KyberHintHandler is IKyberHint, Utils5 {
         (t2eHint, e2tHint) = abi.decode(hint, (bytes, bytes));
     }
 
+    /// @notice Checks that the reserveId and splits arrays must be empty
+    /// @param reserveIds reserve IDs array
+    /// @param splits splits array
+    function checkReserveIdsSplitsIsEmpty(
+        bytes32[] memory reserveIds,
```

Comment:
seems function is over kill for two lines of code.

it makes 2 lines --> 10 lines

so the file as whole is harder to follow.

Reply:
Yes, removed. Similar to earlier comment.